pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -23,10 +23,17 @@ import (
 	"strings"
 
 	nadv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
 	nadutils "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/utils"
 	"github.com/pkg/errors"
 )
 
+// EnforceHostNetwork is a global/package variable, that can be set via the rook-operator-config setting
+// "ROOK_ENFORCE_HOST_NETWORK". when set to "true", it lets rook create all pods with host network enabled. This can be used, for example, to help allow Rook to run in k8s clusters with no CNI, so that host networking is required
+// while setter and getter functions are defined in opcontroller,
+// the variable is defined here in order to avoid import cycles.
```

Comment:
The `./pkg/apis` code is intended to be the public-facing API of Rook. Mentioning internal code here should be avoided.

I would suggest changing this to say something like, "this variable can be defined to force `IsHost()` to return true in such environments," which keeps it focused on the effects it has on the API and not focused on internal code.

Reply:
I just removed the two lines highlighted above, thereby avoiding any mention of internal code. I even avoided to mention `IsHost` in the comment and explained the conceptual effects instead.